FocusSublimeWindow.py

```diff
@@ -4,13 +4,19 @@
 from sublime_plugin import WindowCommand
 
 
+NATIVE_FOCUS_WINDOW = int(sublime.version()) >= 4000
```

Comment:
It seems that it was introduced in BUILD 4113 specifically: https://www.sublimetext.com/download

BUILD 4107 also exists but does not have the method

```suggestion
NATIVE_FOCUS_WINDOW = int(sublime.version()) >= 4113
```

Reply:
4107 is the first (stable) build that has the method.  (And 4067 ist the first *dev* build.)  4107 marks the v4, it's the initial v4. 

![image](https://user-images.githubusercontent.com/8558/219855583-6ba3981c-e91c-46b6-8da5-05874d4aacea.png)